### PR TITLE
Completed Assignment2

### PR DESCRIPTION
**Pull request by Rarkness**
## What did you learn from the changes you have made?
to make sure the path to the file is correct
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I  was not able to create a PR as I didn't create a new branch
## How were these changes tested?
line by line
## Checklist
- [x ] I can confirm that my changes are working as intended

### CODE REVIEW

File: 02_activities/assignments/assignment_1.ipynb
Comment (anjali-deshpande-hub) [inline 02_activities/assignments/assignment_1.ipynb:1]: assignment_1.ipynb file changes should not be part of assignment-2. You probably created assignment-2 branch from assignment-1. Please correct this.

File: 02_activities/assignments/assignment_2.ipynb [hunk 1]
```diff
@@ -1,339 +1,832 @@
...
...8,3,11,5,7,10,5,15,9,16,17,16,3,8,9,8,3,3,9,5,1,6,5,4,2,2,0\\n', '0,1,2,2,2,5,5,1,4,6,3,6,5,9,6,7,4,7,16,7,16,13,9,16,12,6,7,9,10,3,6,4,5,4,6,3,4,3,2,1\\n', '0,1,1,2,3,1,5,1,2,2,5,7,6,6,5,10,6,7,17,13,15,16,17,14,4,4,10,10,10,11,9,9,5,4,4,2,1,0,1,0\\n', '0,1,0,3,2,4,1,1,5,9,10,7,12,10,9,15,12,13,13,6,19,9,10,6,13,5,13,6,7,2,5,5,2,1,1,1,1,3,0,1\\n', '0,1,1,3,1,1,5,5,3,7,2,2,3,12,4,6,8,15,16,16,15,4,14,5,13,10,7,10,6,3,2,3,6,3,3,5,4,3,2,1\\n', '0,0,0,2,2,1,3,4,5,5,6,5,5,12,13,5,7,5,11,15,18,7,9,10,14,12,11,9,10,3,2,9,6,2,2,5,3,0,0,1\\n', '0,0,1,3,3,1,2,1,8,9,2,8,10,3,8,6,10,13,11,17,19,6,4,11,6,12,7,5,5,4,4,8,2,6,6,4,2,2,0,0\\n', '0,1,1,3,4,5,2,1,3,7,9,6,10,5,8,15,11,12,15,6,12,16,6,4,14,3,12,9,6,11,5,8,5,5,6,1,2,1,2,0\\n', '0,0,1,3,1,4,3,6,7,8,5,7,11,3,6,11,6,10,6,19,18,14,6,10,7,9,8,5,8,3,10,2,5,1,5,4,2,1,0,1\\n', '0,1,1,3,3,4,4,6,3,4,9,9,7,6,8,15,12,15,6,11,6,18,5,14,15,12,9,8,3,6,10,6,8,7,2,5,4,3,1,1\\n', '0,1,2,2,4,3,1,4,8,9,5,10,10,3,4,6,7,11,16,6,14,9,11,10,10,7,10,8,8,4,5,8,4,4,5,2,4,1,1,0\\n', '0,0,2,3,4,5,4,6,2,9,7,4,9,10,8,11,16,12,15,17,19,10,18,13,15,11,8,4,7,11,6,7,6,5,1,3,1,0,0,0\\n', '0,1,1,3,1,4,6,2,8,2,10,3,11,9,13,15,5,15,6,10,10,5,14,15,12,7,4,5,11,4,6,9,5,6,1,1,2
```
Comment (anjali-deshpande-hub) [inline 02_activities/assignments/assignment_2.ipynb]: There is incorrect printing inside the loop. You are printing the entire contents list in each iteration of the loop (print(contents)), which is not the intended behaviour.  Instead, you should print each individual line (content).

Comment (anjali-deshpande-hub) [inline 02_activities/assignments/assignment_2.ipynb]: Incorrect printing inside the loop. For every line in the file, you are printing first 5 lines of the file. This is not required.

**Review by anjali-deshpande-hub — CHANGES_REQUESTED** [review]
Please make the above mentioned changes and resubmit

**Comment by Rarkness** [comment]
no idea how to do that

**Comment by Rarkness** [comment]
no idea what you mean or how to do this

File: 02_activities/assignments/assignment_1.ipynb
Comment (anjali-deshpande-hub) [reply 02_activities/assignments/assignment_1.ipynb:1]: Refer to https://uoft-dsi-certificates.slack.com/archives/C080M1JFJ8G/p1733888574840979 to correct this problem

**Comment by anjali-deshpande-hub** [comment]
Please do not close your pull request until the review is complete and it has been approved by learning support. If further changes are requested, simply commit the updates to the same branch; the PR will automatically reflect the changes. There is no need to create a new PR.

**Comment by Rarkness** [comment]
I thought I corrected that. I closed the original pull and
resubmitted it with the correct changes as a new pull (I know don't do that
simply committing is all we have to do).
I had asked the question before doing that but didn't get an answer so I
did what I could.

**Pull request closed without merging**